Assets/AppCenter/Editor/AppCenterPostBuild.cs

```diff
@@ -312,10 +312,9 @@ private static void AddDependenciesToProjectJson(string projectJsonPath)
             return;
         }
         var jsonString = File.ReadAllText(projectJsonPath);
-        jsonString = AddDependencyToProjectJson(jsonString, "Microsoft.NETCore.UniversalWindowsPlatform", "5.2.2");
-        jsonString = AddDependencyToProjectJson(jsonString, "Newtonsoft.Json", "10.0.3");
-        jsonString = AddDependencyToProjectJson(jsonString, "sqlite-net-pcl", "1.3.1");
-        jsonString = AddDependencyToProjectJson(jsonString, "System.Collections.NonGeneric", "4.0.1");
+        jsonString = AddDependencyToProjectJson(jsonString, "Microsoft.NETCore.UniversalWindowsPlatform", "6.2.9");
+        jsonString = AddDependencyToProjectJson(jsonString, "Newtonsoft.Json", "12.0.3");
+        jsonString = AddDependencyToProjectJson(jsonString, "SQLitePCLRaw.bundle_green", "2.0.2");
```

Comment:
> You can't use .NET Standard libraries with .NET scripting backend. You'll have to switch to IL2CPP.

https://forum.unity.com/threads/uwp-app-with-netstandard-libraries.531751/#post-3499618

Does this really work now?

Reply:
Also, this should be in a separate PR

Reply:
> Does this really work now?
This is related to E2E task. Also, we can't fully check this due to another blocking issue which is solved in another task.